src/server.c

```diff
@@ -6343,6 +6347,7 @@ sds genRedisInfoString(const char *section) {
             "mem_total_replication_buffers:%zu\r\n"
             "mem_clients_slaves:%zu\r\n"
             "mem_clients_normal:%zu\r\n"
+            "mem_cluster_links:%zu\r\n"
```

Comment:
this is another interface change

Reply:
Yes. In general memory consumption on the cluster bus has been a blind spot. For example, before we had [this commit](https://github.com/redis/redis/commit/8647addf934c453e933bb5f26ab4950fd5924a04), cluster link buffers never shrunk and could grow unbounded, but we have no way to observe/monitor it while it was happening.

Reply:
by "interface change" i meant it should be listed in the top comment, for the purpose of release notes and approval of reviewers that don't have time to read the code.

i.e. all interface changes are "major decisions" that must have lazy consensus by the core-team.

Reply:
I see. Good to know. Thanks for the info.

Added to the top comment.